returnn/util/basic.py

```diff
@@ -180,6 +180,41 @@ def is_tensorflow_selected(cls):
     return cls.get_selected_engine() == cls.TensorFlow
 
 
+class BehaviorVersion:
+  """
+  Stores the global behavior_version
+
+  The version will be set after the config is defined at __main__.init_config() or Engine.__init__()
+  """
+
+  _behavior_version = 0  # type: typing.Optional[int]  # the selected global behavior_version, default 0 for tests
+  _is_set = False  # type:bool  # store if the behavior version was already set (e.g. re-check in engine init)
```

Comment:
```suggestion
  _is_set = False  # type: bool
```
(I don't like to have spammy documentation which is obvious.)